ftplugin/java.vim

```diff
@@ -0,0 +1,2 @@
+map <silent> <LocalLeader>ra :wa<CR> :RunAllBazelTests<CR>
+map <silent> <LocalLeader>rb :wa<CR> :RunTargetBazelTests<CR>
```

Comment:
Why did this show up? I think it should be gone?

Reply:
Ah, I suspect this is because I moved it in my branch when it was still in the repo, so it appears as a new content. I'll clean that up before merging.

Reply:
Fixed